query-service/src/main/resources/configs/common/application.conf

```diff
@@ -152,7 +152,8 @@ service.config = {
             "API_TRACE.userRole" : "user_role",
             "API_TRACE.userScore" : "user_score",
             "API_TRACE.spaceIds": "space_ids",
-            "API_TRACE.apiExitCalls": "api_exit_calls"
+            "API_TRACE.apiExitCalls": "api_exit_calls",
+            "API_TRACE.apiCalleeInfo" : "api_callee_name_count"
```

Comment:
Had requested in config-bootstrapper PR as `API_TRACE.apiCalleeNameCount` as key for `apiCalleeNameCount`

Reply:
done